benchmarks/spike_adr07_regex.py

```
@@ -0,0 +1,495 @@
+    SPIKE_IRREDUCIBLE_N=1000 python benchmarks/spike_adr07_regex.py
+"""
+
+from __future__ import annotations
+
+import os
+import re
+import statistics
+import sys
+import threading
+import time
+from typing import Callable
```

Flake8 (7.3.0)

[error] 'typing.Callable' imported but unused

(F401)